runner/runner_test.go

```diff
@@ -0,0 +1,72 @@
+package runner_test
```

Comment:
Maybe move this to status_test.go?

Also, does it need to be in a separate package (runner_test instead of runner)?